vital/types/metadata_traits.h

```diff
@@ -95,6 +97,28 @@ class VITAL_EXPORT metadata_traits
    */
   vital_meta_trait_base const& find( vital_metadata_tag tag ) const;
 
+  /// Find traits entry for name.
```

Comment:
Should these follow the current documentation style guidelines? Should the rest of the file be updated?